src/pages/patientView/PatientViewGenePanelModal/PatientViewGenePanelModal.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { GenePanel } from 'shared/api/generated/CBioPortalAPI';
+import { observer } from 'mobx-react';
+import GenePanelModal from 'shared/components/GenePanelModal/GenePanelModal';
+import GenesList from './GenesList';
+
+interface IGenePanelModalProps {
+    genePanelCache: GenePanel;
+    panelName: string;
+    show: boolean;
+    hide: () => void;
```

Comment:
is this a callback that fires when it hides? if so then it should be called `onHide`